### ISSUE TITLE
Update guilds' action filed

### ISSUE THREAD

**Issue by amindadgar** [RnDAO/tc-dbComm#65]
In order to compute the new memberactivities in analytics script we need the `action` field of all guilds to be updated. Each value in the array of `action` is representing the duration for an activity, for example the first entry in it is representing *minimum number of interactions to be active*. 

So, Now, the task is to add two more values to each array of `action`. 
The `action` array should be like: `[1, 1, 1, 4, 3, 5, 5, 4, 3, 3, 2, 2, 1]`
Basically you should update it and add `2, 1` to the end of the existing array.

**Comment by amindadgar** [comment]
I did the task. It is completed now but for it is just for the development server.
For the production, we still needed to do that. I didn't do this for production cause needed approvals (It is not a big task).
 @cyri113

**Comment by amindadgar** [comment]
I can see the action field in the production servers are also updated, we could close this issue.
